openreview/conference/webfield.py

```diff
@@ -323,7 +323,7 @@ def set_reduced_load_page(self, conference_id, invitation, options = {}):
             content = content.replace("var HEADER = {};", "var HEADER = " + json.dumps(header) + ";")
             return self.__update_invitation(invitation, content)
 
-    def set_recruit_page(self, conference_id, invitation, options = {}, reviewers_name='Reviewers'):
+    def set_recruit_page(self, conference_id, invitation, options = {}, reviewers_name='Reviewers', reduced_load=None):
```

Comment:
Can you remove the parameter `reviewers_name` and pass `reduced_load_invitation_id` so no need to build the invitation id here, just pass it as a parameter.